build_scripts/deploy.sh

```diff
@@ -0,0 +1 @@
+curl -dH -X POST "$1"
```

Comment:
I realise I commented tangentially in an earlier review, but this script is able to grab the `$AZURE_DEPLOYMENT_WEBHOOK_URL` variable directly even without it being passed in, so we could switch this from being a positional argument to a named one.

Also separately, not important here, but for good habit suggest adding a shebang to the start of this file